linker/linker_test.go

```diff
@@ -1356,12 +1356,12 @@ func TestSyntheticOneOfCollisions(t *testing.T) {
 	expectedFoo1FirstErrors := []string{
 		`foo2.proto:2:9: symbol "Foo" already defined at foo1.proto:2:9`,
 		`foo2.proto:3:19: symbol "Foo.bar" already defined at foo1.proto:3:19`,
-		`foo2.proto:3:19: symbol "Foo._bar" already defined at foo1.proto:3:19`,
+		`foo2.proto:3:3: symbol "Foo._bar" already defined at foo1.proto:3:3`,
```

Comment:
Also need an update to `nameStart` in `linker/symbols.go`. Its type switch only considers `*ast.OneofNode` but now should be using `ast.OneOfDeclNode` and pulling the name node out of it.